Documentation/ApiOverview/TypoScriptSyntax/Syntax/Conditions.rst

```diff
@@ -274,12 +263,106 @@ from the top level. This is normally done when TypoScript code from
 various records is combined.
 
 
+.. index:: TypoScript; Symfony expression language
 .. _typoscript-syntax-conditions-expression-language:
 
-Custom Conditions With Symfony Expression Language
-==================================================
+Custom conditions with the `Symfony`:pn: expression language
```

Comment:
Just for understanding: shouldn't the tag go before the text to annotate? Similarly to what we do with inline code.

Reply:
Martin Bless suggested of writing it after the text to impeade less with readability. Both is possible, also with other roles like inlide code

Reply:
Well, no regular human will ever read it like this and contributors should be familiar with annotations for code. So I'd vote for putting this in the front for consistency.